server/agents/ReceiptOrchestrator/tools.py

```
@@ -0,0 +1,260 @@
+            'payload': {
+                'genericClasses': [new_class],
+                'genericObjects': [new_object]
+            }
+        }
+
+        # Sign JWT and create save link
+        signer = crypt.RSASigner.from_service_account_file(self.key_file_path)
+        token = jwt.encode(signer, claims).decode('utf-8')
+        return f'https://pay.google.com/gp/v/save/{token}'
+
+import os
```

Ruff (0.12.2)

Redefinition of unused `os` from line 2

Remove definition: `os`

(F811)